src/app/pages/ParaTimesPage/TransactionConfirmation/index.tsx

```diff
@@ -23,20 +23,25 @@ export const TransactionConfirmation = () => {
               ? t('paraTimes.confirmation.receiving', 'into the receiving')
               : t('paraTimes.confirmation.withdrawing', 'from the withdrawing'),
             address: transactionForm.recipient,
-            isEvmc: isEvmcParaTime ? t('paraTimes.common.evmc', '(EVMc)') : '',
+            paratimeType: isEvmcParaTime ? t('paraTimes.common.evmcType', '(EVMc)') : '',
             paraTime: paraTimeName,
             value: transactionForm.amount,
           }}
-          defaults="You are about to transfer <strong>{{value}} ROSE</strong> tokens {{actionType}} wallet on the <strong>{{paraTime}}</strong> {{isEvmc}} ParaTime: <strong>{{address}}</strong>"
+          defaults="You are about to transfer <strong>{{value}} ROSE</strong> tokens {{actionType}} wallet on the <strong>{{paraTime}}</strong> {{paratimeType}} ParaTime: <strong>{{address}}</strong>"
```

Comment:
`<strong>{{value}} ROSE</strong>`

should use selectTicker, so it displays TEST on testnet